### PR TITLE
[MOB-4936] Fix issue with discrepancy in iframe heights and refactor mock data to test cached messages

### PR DESCRIPTION
**Pull request by pauljung14**
## JIRA Ticket(s) if any

* [MOB-4936](https://iterable.atlassian.net/browse/MOB-4936)

## Description

Even though there isn't a scrollbar, the iframe height differs from the actual iframe document's html's height. The extra 10px of adjustment fixes issue for top/bottom/center layouts (not applicable in full layout).

Also refactors mock data to make it more usable with cache testing.

## Test Steps

Paint inapp to screen in top/bottom/center layouts.
Place mouse over the message and scroll.
Message should not move.

### CODE REVIEW

File: src/__data__/inAppMessages.ts [hunk 1]
```diff
@@ -1,165 +1,172 @@
 import { InAppMessage } from '../inapp/types';
```
Comment (pauljung14) [inline src/__data__/inAppMessages.ts:2]: The changes to this file are minimal:
* Pulled each inapp message out to its own named variable for ease of knowing differences
* Updated "normal" message to not have expired (inevitably for now) timestamp
* Set up one that can be swapped out with the commented "previouslyCachedMessage" if a mock for the message delta call is needed

File: src/inapp/utils.ts [hunk 2]
```diff
@@ -429,9 +428,11 @@ export const paintIFrame = (
             For web in-app messages created with WYSIWYG editor,
             there is 8px margin all around the iframe document body.
             Add 16px to the iframe height to eliminate scrollbar.
+            Add 10px to eliminate discrepancy between the height
+            of the document html and that of the iframe itself.
           */
           iframe.style.height =
-            ((iframeHeight && iframeHeight + 16) || 0) + 'px';
+            ((iframeHeight && iframeHeight + 26) || 0) + 'px';
```
Comment (pauljung14) [inline src/inapp/utils.ts:435]: Even though there isn't a scrollbar, the iframe height differs from the actual iframe document's html's height. The extra 10px of adjustment fixes issue for top/bottom/center layouts (not applicable in full layout). Included brief comment above too.

File: src/__data__/inAppMessages.ts [hunk 3]
```diff
@@ -1,165 +1,172 @@
...
-          displayOption: 'AutoExpand'
-        },
-        left: {
-          percentage: 0
-        }
+const normalMessage: InAppMessage = {
+  messageId: 'normalMessage!',
+  campaignId: 2642276,
+  createdAt: 1627060811283,
+  expiresAt: 1670355237175,
```
Comment (pauljung14) [inline src/__data__/inAppMessages.ts:7]: updated value

File: src/__data__/inAppMessages.ts [hunk 4]
```diff
@@ -1,165 +1,172 @@
...
+  },
+  priorityLevel: 300.5,
+  read: false
+};
+
+const expiredMessage: InAppMessage = {
+  messageId: 'expiredMessage!',
+  campaignId: 2641072,
+  createdAt: 1627320168381,
+  expiresAt: 1635096168381,
```
Comment (pauljung14) [inline src/__data__/inAppMessages.ts:58]: original, still-expired value

File: src/__data__/inAppMessages.ts [hunk 5]
```diff
@@ -1,165 +1,172 @@
...
+  saveToInbox: false,
+  priorityLevel: 300.5,
+  read: false
+};
+
+const previouslyCachedMessage: InAppMessage = {
+  messageId: 'previouslyCachedMessage!',
+  campaignId: 2632531,
+  createdAt: 1627332441416,
+  expiresAt: 1670355237175,
```
Comment (pauljung14) [inline src/__data__/inAppMessages.ts:115]: updated value (expired messages get immediately removed from the cache so should test with unexpired message)

**Review by mprew97 — APPROVED** [review]

**Pull request merged**